docs/test_scripts/test_simple_distributed.py

```diff
@@ -0,0 +1,85 @@
+# [imports]
+import torch
+from torch.nn.parallel import DistributedDataParallel
+
+import modulus
+from modulus.datapipes.benchmarks.darcy import Darcy2D
+from modulus.distributed import DistributedManager
+from modulus.metrics.general.mse import mse
+from modulus.models.fno.fno import FNO
+from modulus.utils import StaticCaptureTraining
+
+# [imports]
+
+# [code]
+
+
+def main():
+
+    DistributedManager.initialize()
+    dist = DistributedManager()
+
+    normaliser = {
+        "permeability": (1.25, 0.75),
+        "darcy": (4.52e-2, 2.79e-2),
+    }
+    dataloader = Darcy2D(
+        resolution=256, batch_size=64, nr_permeability_freq=5, normaliser=normaliser
+    )
+    model = FNO(
+        in_channels=1,
+        out_channels=1,
+        decoder_layers=1,
+        decoder_layer_size=32,
+        dimension=2,
+        latent_channels=32,
+        num_fno_layers=4,
+        num_fno_modes=12,
+        padding=5,
+    ).to("cuda")
+
+    if dist.distributed:
+        ddps = torch.cuda.Stream()
+        with torch.cuda.stream(ddps):
+            model = DistributedDataParallel(
+                model,
+                device_ids=[dist.local_rank],  # Set the device_id to be
+                # the local rank of this process on
```

Comment:
Indenting here is off

Reply:
Yeah I saw this, Actually `ruff` is the culprit here, haha! Let me fix this nevertheless

Reply:
fixed